Source/SwipeTableViewCell.swift

```diff
@@ -99,17 +102,19 @@ open class SwipeTableViewCell: UITableViewCell {
     override open func didMoveToSuperview() {
         super.didMoveToSuperview()
         
-        var view: UIView = self
-        while let superview = view.superview {
-            view = superview
-            
-            if let tableView = view as? UITableView {
-                self.tableView = tableView
+        if #available(iOS 10, *) {
```

Comment:
Considering there's a separate pull request to provide iOS 9 support, it would probably be better to check if the delegate is set instead:  

```swift
guard delegate != nil else { return }
```